core/domain/feedback_jobs_continuous_test.py

```diff
@@ -111,14 +112,17 @@ def test_get_thread_analytics_multi(self):
             thread = feedback_models.FeedbackThreadModel.create(
                 exp_id_1, thread_id)
             thread.exploration_id = exp_id_1
+            event_services.FeedbackThreadCreatedEventHandler.record(
```

Comment:
No -- I meant, you should create the thread using the methods in feedback_services, and not touch the model directly at all. (I know that the other tests here do the latter, but you're actually checking a different thing, which is whether the realtime layer works properly or not.)

The trouble with doing it this way, where you make the calls manually, is that it introduces skew. If, say, a developer made a change that accidentally removed the event firing in the main codebase, these tests would still pass.